src/utils/ipfs.js

```diff
@@ -51,6 +57,15 @@ export const saveToIpfs = (files, cb, imgCb) => {
 
 export const uploadToIpfs = async files => {
   const res = await ipfs.add(files, { progress: (prog) => console.log(`received: ${prog}`) })
+  const miniUpload = await ipfsMini.add(files)
+  console.log({miniUpload})
+  return `ipfs/${res[0].hash}`
+}
+
+export const uploadToIpfsGateway = async files => {
```

Comment:
`uploadToIpfs ` and `uploadToIpfsGateway ` do the same thing. Do we need both?

Reply:
That was an oversight, uploadToIpfs should not upload to the the gateway. Pushed a commit fixing that.